changelog/unreleased/issue-4523

```diff
@@ -0,0 +1,16 @@
+Bugfix: Update zstd library to fix possible data corruption at max. compression
+
+In restic 0.16.0, backups using maximum compression could result in data
+corruption. The "auto" compression level (used by default) is not affected!
```

Comment:
I suggest `.. could result in data corruption due to a bug in the library used for compressing data. Please note that the "auto" compression level (used by default) is not affected!`.

Reply:
I've applied the suggestions.